dev-server/documents/html/cross-origin-iframe.mustache

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Cross-origin guest iframe</title>
+    <style>
+      body {
+        font-family: sans-serif;
+      }
+      iframe {
+        width: 75%;
+        height: 300px;
+        resize: both;
+        overflow: auto;
+      }
+    </style>
+
+  </head>
+  <body>
+    <h1>Cross-origin guest iframe</h1>
+    <p>
+      The cross-origin iframe below should behave like a normal guest iframe. It should
+      be possible to select text and add annotations.
+    </p>
+    <iframe></iframe>
+    <script>
+      // Set a different port for the iframe `src` property to make it cross-origin.
+      let port = new URL(document.location).port === '3000' ? '3002' : '3000';
+      let url = '//localhost:' + port + '/document/burns-embedded';
+      document.querySelector('iframe').setAttribute('src', url);
```

Comment:
It is not the most terse JS because JS is not transpiled and I wanted to make it compatible with all the browsers we support. 
Related to #2900